### ISSUE TITLE
mono iOS export doesn’t create .ipa

### ISSUE THREAD

**Issue by samtsejerome** [godotengine/godot#45421]
**Godot version:**
3.2.3 official build.mono

**OS/device including version:**
macbook pro running catalina 10.15.6

**Issue description:**
I've tried this a few times with different projects and always get the same result:

steps to reproduce: 
1. create a simple project  with a viewport that should  fit any modern  iphone display.
2. select export... from the project menu.
3. select the ios(Runnable) preset
4. enter a valid team id, and bundle id
5. select the  export path to a location outside of the project.
6. select all the required icons, and screens as per the documentation.
7. click the export Project button, enter an ipa filename and click save.

expected results:
an ipa is created.

actual results:
a progress indicator is displayed that gets stuck at 60% for an impractically long time. (20 mins + )
<img width="518" alt="exporrtprogress-sshot" src="https://user-images.githubusercontent.com/68197715/105641075-166a0500-5e7a-11eb-9be6-0296146048a0.png">

when it finally finishes, no .ipa file has been created!

**Minimal reproduction project:**

[csharptest_project3.zip](https://github.com/godotengine/godot/files/5863070/csharptest_project3.zip)

**Comment by naithar** [comment]
MRP doesn't have required iOS icons and LaunchScreen images that's needed for correct export. 
After providing them (icons and images) I've managed to successfully export a project and received `.ipa` file. XCode project also successfully launches and compiles.

Logged errors like this one indicate that you are missing a file that export mechanism requires.
```
 core/os/dir_access.cpp:298 - Failed to open res://ios_icons/landscape Launch screens/2436x1125.png
 platform/iphone/export/export.cpp:673 - Failed to export loading screen (landscape_launch_screens/iphone_2436x1125) from path 'res://ios_icons/landscape Launch screens/2436x1125.png'.
 editor/project_export.cpp:984 - Failed to export the project for platform 'iOS'.
```

**Comment by Tekuzo** [comment]
I have also experienced this issue. My project had the icons set properly but did not have LaunchScreen Images, I had left those blank because I had scripted an entire scene to handle this. The export worked in 3.2.3.

**Comment by Calinou** [comment]
@Tekuzo To clarify, did you test this both on 3.2.3 and 3.2.4rc3 or just 3.2.3?

**Comment by naithar** [comment]
> I have also experienced this issue.

Have you tried manually building exported Xcode project?

**Comment by neikeq** [comment]
This is likely the C# part of the export process failed, but Godot's export process continues. The export process should stop if the C# exporter fails, but that's a separate thing. Run Godot from the terminal and check the output. Right now there are many issues exporting to iOS with C# that I've just fixed but can't be merged in the 3.x branch yet.

**Comment by neikeq** [comment]
Assuming fixed by #49248. Let me know if that's not the case (the fix should be included in the next release).
